src/utils/numbers.js

```diff
@@ -46,3 +47,7 @@ export const calculateMinPayment = (interest) => {
   const interestWithMargin = interest.mul(10010).div(10000);
   return interestWithMargin.lt(floor) ? floor : interestWithMargin;
 };
+
+export const calculateInterestRate = (borrowRatePerBlock, chainId) => {
+  return format(borrowRatePerBlock.mul(BlocksPerYear[chainId]).toString());
```

Comment:
Not: try and keep these types of function returning bigNumber and then do the formatting when we render.